vpr/src/device/rr_graph_view.h

```diff
@@ -249,6 +249,36 @@ class RRGraphView {
         return node_storage_.node_side_string(node);
     }
 
+    /** @brief Get the number of configurable edges. This function is inlined for runtime optimization. */
+    inline t_edge_size num_configurable_edges(RRNodeId node) const {
+        return node_storage_.num_configurable_edges(node);
+    }
+
+    /** @brief Get the number of non-configurable edges. This function is inlined for runtime optimization. */
```

Comment:
TODO: Add more comments about configurable and non-configurable edges (what they are).

Reply:
Sure, will do that.